src/mot/serving/utils.py

```diff
@@ -46,12 +51,18 @@ def handle_post_request(upload_folder=UPLOAD_FOLDER) -> Dict[str, np.array]:
     data = json.loads(request.data.decode("utf-8"))
     if "image" in data:
         image = np.array(data["image"])
-        return localizer_tensorflow_serving_inference(image, SERVING_URL)
+        outputs = localizer_tensorflow_serving_inference(image, SERVING_URL)
+        detected_trash = []
+        for box,label,score in zip(outputs["output/boxes:0"],outputs["output/labels:0"],outputs["output/scores:0"]):
+            trash_json = {"box":[x for x in box], "label":cfg.DATA.CLASS_NAMES[label], "score":score}
+            detected_trash.append(trash_json)
+        return {"detected_trash": detected_trash}
+
     elif "video" in data:
         raise NotImplementedError("video")
 
 
-def handle_file(file: FileStorage, upload_folder=UPLOAD_FOLDER) -> Dict[str, np.array]:
+def handle_file(file: FileStorage, upload_folder=UPLOAD_FOLDER, fps=2) -> Dict[str, np.array]:
```

Comment:
upload_folder is always set to tmp. Usually if the file needs to be added in a tmp folder the best practice is to use: ` tempfile.mkdtemp` as a context manager.

e.g:

```
with  tempfile.mkdtemp() as upload_folder:
   # do something
```